sound/hybrid_font.h

```diff
@@ -186,6 +191,12 @@ class HybridFont : public SaberBase {
     // Radians per second per second
     float rss = sqrtf(gyro_slope.z * gyro_slope.z + gyro_slope.y * gyro_slope.y) * (M_PI / 180);
     float swing_speed = fusor.swing_speed();
+    uint32_t now = micros();
+    uint32_t delta_micros = now - last_swing_micros_;
+    last_swing_micros_ = now;
+    if (delta_micros > 1000000) delta_micros = 1;
+    float delta = delta_micros * 0.000001;
```

Comment:
It occurs to me that line 198 & 199 can be moved into the if statement below.
We don't need to do those calculations when we're not swinging.